src/oiiotool/oiiotool.h

```diff
@@ -171,13 +171,15 @@ class Oiiotool {
     // Expand substitution expressions in string str. Expressions are
     // enclosed in braces: {...}. An expression consists of:
     //   * a numeric constant ("42" or "3.14")
-    //   * simple math expr+expr, expr-expr, expr*expr, expr/expr, evaluated
-    //     left-to-right (no operator precedence).
+    //   * arbitrary math using operators +, -, *, / and parentheses
+    //     (order of operations is respected).
     //   * IMG[n].metadata for the metadata of an image. The 'n' may be an
     //     image name, or an integer giving stack position (for example,
     //     "IMG[0]" is the top of the stack; also "TOP" is a synonym). The
     //     metadata can be any of the usual named metadata from the image's
     //     spec, such as "width", "ImageDescription", etc.
+    //   * string constants (space-delimited or double-quoted)
+    //   * string concatenation using the + operator
```

Comment:
You removed string concatenation with +, right? Probably should remove this line from the comment.